### PR TITLE
Specification Requirement -  Markdown

### PR DESCRIPTION
**Pull request by Nethmaw**
Specification requirement document based on the conduit website written in a markdown file

### CODE REVIEW

File: specificationRequirements
Comment (dannycrone) [inline specificationRequirements:1]: It looks like this file does not have extension - I would expect it to have been called requirements-specification.md

Comment (dannycrone) [inline specificationRequirements:1]: Also, I was expecting this to be in a directory of its own - e.g. docs

**Review by dannycrone — CHANGES_REQUESTED** [review]
I have left a few comments on here which need to be resolved before I can approve them

File: specificationRequirements
Comment (Nethmaw) [reply specificationRequirements:1]: The naming has been changed

Comment (Nethmaw) [reply specificationRequirements:1]: DOCS folder created

File: prisma/db.sqlite
[the remark this answers is not in the document]
Comment (Nethmaw) [reply prisma/db.sqlite:1]: This file will be deleted

[the remark this answers is not in the document]
Comment (dannycrone) [reply prisma/db.sqlite:1]: > This file will be deleted

I don't think you want to delete this file, I think you just want to undo the changes to it

File: specificationRequirements
Comment (dannycrone) [reply specificationRequirements:1]: Cool with that

File: DOCS/specificationRequirements.txt
Comment (dannycrone) [inline DOCS/specificationRequirements.txt:1]: Why do you need this file

File: prisma/db.sqlite
[the remark this answers is not in the document]
Comment (dannycrone) [reply prisma/db.sqlite:1]: I guess it is fine

File: DOCS/specificationRequirements.txt
Comment (dannycrone) [reply DOCS/specificationRequirements.txt:1]: I've fixed this up with commit 9a0855fbf147d69e7b8cefae6fdbe6bc3b0e4b20

File: DOCS/requirements-specification.md
[the remark this answers is not in the document]
Comment (dannycrone) [reply DOCS/requirements-specification.md:1]: I resolved this with 0f908d31c77eb6ea591a0ee8080549349f5cdf39 followed by ba1f9082a57aaeda0fd42d8d2aacd8daeae82777

**Review by dannycrone — APPROVED** [review]

**Pull request merged**